source/testing/index.md

```diff
@@ -1,79 +1,149 @@
+This will ensure that the features within a project are basically functional,
+and ensure that the core features of a project have not regressed, and that the project's goals are being met.
 
-In the example scenario above, some acceptance tests one might write are:
+In the example scenario above, some of these tests you might write are:
 
 * A user is able to log in via the login form.
 * A user is able to create a blog post.
 * After saving a new post successfully, a user is then shown the list of prior posts.
 * A visitor does not have access to the admin panel.
 
-### Unit Tests
+For tests that require a full application setup we can use the `setupApplicationTest` helper.
+You can read more about how to create these kind of tests in the [Application tests] section.
 
-Unit tests are used to test isolated chunks of functionality, or "units".
-They can be written against any isolated application logic.
+### Rendering Tests
 
-Some specific examples of units tests are:
+Some tests serve as a middle ground between those tests, which require a full application setup and those,
+which interact with specific code algorithms on a micro level.
+Oftentimes these tests will verify interactions between various parts of the application,
+such as behavior between UI controls.
+Typically they require an UI to be rendered to be able to execute user interactions in the testing scenario.
+They are valuable in ensuring data and actions are properly passed between different parts of the system,
+and provide confidence that parts of the system will work within the application under multiple scenarios.
 
-* A fullname attribute is computed which combines the `firstName` and `lastName` attributes.
-* The serializer properly converts the blog request payload into a blog post model object.
-* Blog dates are properly formatted.
-
-### Integration Tests
-
-Integration tests serve as a middle ground between acceptance tests, which only interact
-with the full system through user endpoints, and unit tests, which interact with specific
-code algorithms on a micro level. Integration tests verify interactions between various
-parts of the application, such as behavior between UI controls.  They are valuable
-in ensuring data and actions are properly passed between different parts of the system, and
-provide confidence that parts of the system will work within the application under multiple
-scenarios.
-
-It is recommended that components and helpers be tested with integration tests because the component
-interacts with the system in the same way that it will within the context of the application,
+Rendering tests are especially useful for verifying the correct behavior of components and helpers.
+The component or helper interacts with the system in the same way that it will within the context of the application,
 including being rendered from a template and receiving Ember's lifecycle hooks.
 
-Examples of integration tests are:
+Examples of these mid-complex tests are:
```

Comment:
Is mid-complex a specific term known in generic testing/new Ember testing API? I found this a little confusing, like it was a different type of test.

Reply:
No, it isn’t a term related to the changes in testing API

Reply:
Thanks Robert. Let's avoid that label then.